src/main/java/org/apache/cassandra/sidecar/db/RestoreJob.java

```diff
@@ -76,10 +75,13 @@ public static RestoreJob from(@NotNull Row row) throws DataObjectMappingExceptio
                .jobSecrets(decodeJobSecrets(row.getBytes("blob_secrets")))
                .expireAt(row.getTimestamp("expire_at"))
                .sstableImportOptions(decodeSSTableImportOptions(row.getBytes("import_options")));
+        String consistencyLevel = row.getString("consistency_level");
+        builder.manager(consistencyLevel == null ? Manager.SPARK : Manager.SIDECAR)
```

Comment:
Took me a moment to figure out why consistencyLevel would be null if the manager is SPARK - can we document why this is the case here please?